enhancements/console/quick-starts.md

```diff
@@ -151,6 +155,28 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The Quick Start
+CRD contains an optional `console.openshift.io/lang` label to specify what
+language the Quick Start is in. Quick Starts without this label will be treated
+as English Quick Starts. OpenShift can display the correct
+Quick Starts for each language by comparing the value of the
+`console.openshift.io/lang` label to the currently displayed language in OpenShift.
+See the [internationalization enhancement proposal](https://github.com/openshift/enhancements/blob/master/enhancements/console/internationalization.md) for more information.
+This approach will not handle languages we don't offer support for; if Quick Starts
+in non-supported languages become a need, we will need to modify this approach.
+
+The language label must be a two-letter
+[ISO 639-2 Language Code](https://www.loc.gov/standards/iso639-2/php/code_list.php))
+(i.e. "en" for English) or an ISO language code hyphenated with an alpha-2
+[ISO-3166 Country Code](https://www.iso.org/iso-3166-country-codes.html)
+(i.e. "en-us").
+
+The language is also appended to the name of non-English Quick Starts so we can avoid showing
+duplicates of the same Quick Start content in different languages. The format is
+my-quick-start for English and my-quick-start-{language code} for other languages.
```

Comment:
Can you show an example YAML snippet with the metadata changes for another language here?